Python/bytecodes.c

```diff
@@ -1441,13 +1441,10 @@ dummy_func(
 
         // stack effect: (__array[oparg] -- __0)
         inst(BUILD_LIST) {
-            PyObject *list =  PyList_New(oparg);
+            STACK_SHRINK(oparg);
```

Comment:
This seems to change behaviour in case list creation fails (the items on the stack would be leaked I think).

Reply:
Ah I see, you deal with this in the new function.